apps/openchallenges/mcp-server/project.json

```diff
@@ -72,6 +72,13 @@
         "command": "trivy image ghcr.io/sage-bionetworks/{projectName}:local --quiet",
         "color": true
       }
+    },
+    "inspect": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "mcp-inspector node build/index.js",
```

Comment:
The port the inspector server listen to is dynamic, so the instructions in the README may not be accurate. More importantly, dynamic ports pollute the ports listed under the ports section of VS Code. Check if there is a way to enforce the server port of the inspector.

Reply:
the ports for both client and server of inspector are now hardcoded in the task